files/zh-cn/web/api/event/index.md

```diff
@@ -82,72 +75,58 @@ slug: Web/API/Event
-  - : 表示事件流正被处理到了哪个阶段。
-- {{domxref("Event.explicitOriginalTarget")}} {{non-standard_inline}} {{readonlyinline}}
-  - : 事件的明确（explicit）原始目标（Mozilla 专有属性）。
-- {{domxref("Event.originalTarget")}} {{non-standard_inline}} {{readonlyinline}}
-  - : 重设目标前的事件原始目标（Mozilla 专有属性）。
-- {{domxref("Event.returnValue")}}
-  - : 旧版 Internet Explorer 引入的一个非标准历史属性，为保证依赖此属性的网页正常运作，此属性最终被收入规范。可用 {{domxref("Event.preventDefault()")}} 与 {{domxref("Event.defaultPrevented")}} 代替，但由于已进入规范，也可以使用此属性。
-- {{domxref("Event.srcElement")}} {{non-standard_inline}}
-  - : 旧版 Internet Explorer 对 {{domxref("Event.target")}} 的非标准别称。出于兼容原因，一些其他浏览器也支持此别称。
-- {{domxref("Event.target")}} {{readonlyinline}}
-  - : 对事件原始目标的引用，这里的原始目标指最初派发（dispatch）事件时指定的目标。
-- {{domxref("Event.timeStamp")}} {{readonlyinline}}
+- {{domxref("Event.eventPhase")}} {{ReadOnlyInline}}
+  - : 指示正在处理的事件流阶段。它是以下数字之一：`NONE`、`CAPTURING_PHASE`、`AT_TARGET`、`BUBBLING_PHASE`。
+- {{domxref("Event.isTrusted")}} {{ReadOnlyInline}}
+  - : 表示事件是由浏览器（例如用户点击）发起的，还是由脚本发起的（例如使用事件创建方法）。
+- {{domxref("Event.target")}} {{ReadOnlyInline}}
+  - : 对最初分发事件的对象的引用。
+- {{domxref("Event.timeStamp")}} {{ReadOnlyInline}}
   - : 事件创建时的时间戳（精度为毫秒）。按照规范，这个时间戳是 Unix 纪元起经过的毫秒数，但实际上，在不同的浏览器中，对此时间戳的定义也有所不同。另外，规范正在将其修改为 {{domxref("DOMHighResTimeStamp")}}。
-- {{domxref("Event.type")}} {{readonlyinline}}
+- {{domxref("Event.type")}} {{ReadOnlyInline}}
   - : 事件的类型，不区分大小写。
-- {{domxref("Event.isTrusted")}} {{readonlyinline}}
-  - : 表示事件是由浏览器（例如用户点击）发起的，还是由脚本（使用事件创建方法，例如 {{domxref("Event.initEvent")}}）发出的。
 
-### 废弃属性
+### 遗留属性和非标准属性
 
-- {{domxref("Event.scoped")}} {{readonlyinline}} {{Deprecated_Inline}}
-  - : 已废弃，使用 {{domxref("Event.composed")}} 代替此属性。
-    一个{{jsxref("Boolean", "布尔值")}}，表示给定的事件是否会穿过 Shadow DOM，进入到标准 DOM 中。
+- {{domxref("Event.cancelBubble")}} {{deprecated_inline}}
+  - : {{domxref("Event.stopPropagation()")}} 的历史别名，应该改用。在事件处理程序返回之前将其值设置为 `true` 可以阻止事件传播。
+- {{domxref("Event.explicitOriginalTarget")}} {{non-standard_inline}} {{ReadOnlyInline}}
+  - : 事件的明确原始目标。
+- {{domxref("Event.originalTarget")}} {{non-standard_inline}} {{ReadOnlyInline}}
+  - : 重定向目标前的事件原始目标。
+- {{domxref("Event.returnValue")}} {{deprecated_inline}}
+  - 为了确保现有网站继续工作，而仍然支持的历史属性。请改用 {{domxref("Event.preventDefault()")}} 和 {{domxref("Event.defaultPrevented")}}。
+- {{domxref("Event.composed", "Event.scoped")}} {{ReadOnlyInline}} {{deprecated_inline}}
+  - : A boolean value indicating whether the given event will bubble across through the shadow root into the standard DOM. Use {{domxref("Event.composed", "composed")}} instead.
+  - : 一个布尔值，指示给定事件是否会穿过影子根节点冒泡到标准 DOM。请改用 {{domxref("Event.composed", "composed")}}。
 
-## 方法
+## 实例方法
 
-- {{domxref("Event.createEvent()")}} {{deprecated_inline}}
-  - : 创建一个新事件，如果使用此方法创建事件，则必须调用其自身的 `initEvent()` 方法，对其进行初始化。
 - {{domxref("Event.composedPath()")}}
   - : 返回事件的路径（将在该对象上调用监听器）。如果阴影根节点 (shadow root) 创建时 {{domxref("ShadowRoot.mode")}} 值为 closed，那么路径不会包括该根节点下阴影树 (shadow tree) 的节点。
```

Comment:
see: https://github.com/mdn/translated-content/pull/13432#discussion_r1208745293

Reply:
resolved in #13432, the translation is suitable here ;)